tutorials/Coordinates/Coordinates.ipynb

```diff
@@ -0,0 +1,781 @@
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "impix = 1024\n",
+      "imsize = 12*u.arcmin\n",
+      "cutoutbaseurl = 'http://skyservice.pha.jhu.edu/DR12/ImgCutout/getjpeg.aspx'\n",
+      "cutouturl = cutoutbaseurl + '?' + urllib.urlencode(dict(ra=hcg7_center.ra.deg, dec=hcg7_center.dec.deg, \n",
+      "                                                        width=impix, height=impix, \n",
+      "                                                        scale=imsize.to(u.arcsec).value/impix))\n",
+      "urllib.urlretrieve(cutouturl, 'HCG7_SDSS_cutout.jpg')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "This requires an internet connection, but if it failed don't worry: the file is included in the repository so you can just pretend it suceeded.  Now lets take a look at the image."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "IPython.display.Image('HCG7_SDSS_cutout.jpg')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Very pretty!  This shows HCG7 is in the SDSS imaging survey, so we can use the cells below to download catalogs of objects directly from the [Sloan Digitial Sky Survey](http://www.sdss.org/) SQL database using the [astroquery](https://astroquery.readthedocs.org) affiliated package. This will require an internet connection and a working install of astroquery (If you don't have these you can just skip down two cells, because the data files are provided with the repository). Depending on your version of astroquery it might also issue a warning, which you should be able to safely ignore."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "from astroquery.sdss import SDSS\n",
+      "sdss = SDSS.query_region(coordinates=hcg7_center, radius=20*u.arcmin, spectro=True, photoobj_fields=['ra','dec','u','g','r','i','z'])"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "`astroquery` gives us back `astropy.table.Table` objects, which we can write to disk so that we don't have to run the query every time we want to use it."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "sdss.write('HCG7_SDSS_photo.dat', format='ascii')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Now we simply load them back from disk. You can skip this step as long as your astroquery call suceeded, as the tables will be identical given that you just wrote them."
```

Comment:
Ah, gotcha.  I'll re-word to clarify this - looking back on it I see it's pretty unclear.  You don't *have* to write it - rather I'm including it because you *can*, and by doing it in this order, users without internet access will only see a failure at the query and here rather than throughout the whole rest of the section.